vhdbuilder/packer/install-dependencies.sh

```diff
@@ -485,9 +485,11 @@ capture_benchmark "download_gpu_device_plugin"
 
 mkdir -p /var/log/azure/Microsoft.Azure.Extensions.CustomScript/events
 
-systemctlEnableAndStart cgroup-memory-telemetry.timer || exit 1
-systemctl enable cgroup-memory-telemetry.service || exit 1
-systemctl restart cgroup-memory-telemetry.service
+if [ ! isMarinerOrAzureLinux "$OS" ]; then
```

Comment:
can we add a comment for why we're no longer doing this on azure linux?

Reply:
Comment added.